### PR TITLE
Update e2e-guide.md

### PR DESCRIPTION
**Pull request by noobwei**

### CODE REVIEW

**Review by hzxuzhonghu — APPROVED** [review]

**Comment by hzxuzhonghu** [comment]
defer to @YaoZengzeng

File: content/en/docs/developer/e2e-guide.md [hunk 1]
```diff
@@ -46,6 +46,16 @@ When testing locally, you may want to skip some setup steps to save time, especi
 - `--skip-setup`: Skips deploying the Kubernetes cluster, Istio, and Kmesh.
 - `--only-run-tests`: Skips all other steps and focuses only on deploying test applications and running E2E tests.
 
+You might need different kinds of tests.
```
Comment (YaoZengzeng) [inline content/en/docs/developer/e2e-guide.md:49]: List all flags first and then explain them in the following example.

File: content/en/docs/developer/e2e-guide.md [hunk 2]
```diff
@@ -46,6 +46,16 @@ When testing locally, you may want to skip some setup steps to save time, especi
 - `--skip-setup`: Skips deploying the Kubernetes cluster, Istio, and Kmesh.
 - `--only-run-tests`: Skips all other steps and focuses only on deploying test applications and running E2E tests.
 
+You might need different kinds of tests. 
+- `--cluster`: Allows specifying a pre-existing KinD cluster by name.
+- `--ipv6`: Enables creating a KinD cluster with IPv6 networking.
```
Comment (YaoZengzeng) [inline content/en/docs/developer/e2e-guide.md:51]:
```suggestion
- `--ipv6`: Enables creating a KinD cluster with IPv6 networking and run E2E tests on it.
```

File: content/en/docs/developer/e2e-guide.md [hunk 3]
```diff
@@ -62,6 +72,27 @@ When testing locally, you may want to skip some setup steps to save time, especi
   ./test/e2e/run_test.sh --only-run-tests
   ```
 
-## How to add test case
+- Specifying a Pre-existing KinD Cluster:
+
+  ```bash
+  ./test/e2e/run_test.sh --cluster <KinD-Cluster-Name>
+  ```
+- Cleanup the KinD Cluster or Docker Registry After Tests
```
Comment (YaoZengzeng) [inline content/en/docs/developer/e2e-guide.md:80]:
```suggestion
- Cleanup the KinD Cluster and Docker Registry After Tests
```

File: content/en/docs/developer/e2e-guide.md [hunk 4]
```diff
@@ -62,6 +72,27 @@ When testing locally, you may want to skip some setup steps to save time, especi
...
+  ./test/e2e/run_test.sh --cleanup
+  ```
 
-TODO
+You can directly use go test command-line arguments when running the tests. For example, you can filter specific tests, or control other aspects of the testing process directly from the command line.
+
+- Selecting Specific Test Cases:
+
+  ```bash
+  ./test/e2e/run_test.sh --select-cases TestSpecificCase
```
Comment (YaoZengzeng) [inline content/en/docs/developer/e2e-guide.md:91]:
```suggestion
  ./test/e2e/run_test.sh --only-run-tests -run "TestServices"
```

`--select-cases` has been deleted, use `go test` flag directly.

File: content/en/docs/developer/e2e-guide.md [hunk 5]
```diff
@@ -62,6 +72,27 @@ When testing locally, you may want to skip some setup steps to save time, especi
...
+- Specifying a Pre-existing KinD Cluster:
+
+  ```bash
+  ./test/e2e/run_test.sh --cluster <KinD-Cluster-Name>
+  ```
+- Cleanup the KinD Cluster or Docker Registry After Tests
+
+  ```bash
+  ./test/e2e/run_test.sh --cleanup
+  ```
```
Comment (YaoZengzeng) [inline content/en/docs/developer/e2e-guide.md:84]: Add cases about ipv6 and `skip-cleanup-apps`

File: content/en/docs/developer/e2e-guide.md [hunk 6]
```diff
@@ -62,6 +72,27 @@ When testing locally, you may want to skip some setup steps to save time, especi
...
-TODO
+You can directly use go test command-line arguments when running the tests. For example, you can filter specific tests, or control other aspects of the testing process directly from the command line.
+
+- Selecting Specific Test Cases:
+
+  ```bash
+  ./test/e2e/run_test.sh --select-cases TestSpecificCase
+  ```
+
+- Controlling Test Verbosity
```
Comment (YaoZengzeng) [inline content/en/docs/developer/e2e-guide.md:94]: This example is tedious, maybe could add an example that show how to repeat test cases multiple times.

**Pull request merged**